cleanlab_cli/dataset/upload_helpers.py

```diff
@@ -433,3 +443,10 @@ def save_feedback(feedback, filename):
     progress(f"Writing issues to {filename}...")
     dump_json(filename, feedback)
     success("Saved.\n")
+
+
+def extract_float_value(column_value):
+    float_regex_pattern = "[-+]? (?: (?: \d* \. \d+ ) | (?: \d+ \.? ) )(?: [Ee] [+-]? \d+ ) ?"
```

Comment:
Are there supposed to be spaces in the pattern? Tested in regex101.com and it seems like that spaces are not ignored (i.e. this pattern requires that the float contain spaces)